### ISSUE TITLE
Inconsistent data type for GPA in grades Json string

### ISSUE THREAD

**Issue by VinayGupta23** [aneesh-neelam/VITacademics#77]
Regarding the Json string for grades parsing,
The data type for the `"gpa"` key in `"semester_wise"` grades is currently `string`, whereas that of `"cgpa"` is `float`.
However, data types for both gpa and cgpa should be `float'. A portion of the Json is attached for reference:

``` Json
"semester_wise": {
        "2013-11": {
            "credits": 21,
            "gpa": "8.28"
        },
        "2014-05": {
            "credits": 21,
            "gpa": "8.57"
        },
        "2014-11": {
            "credits": 24,
            "gpa": "8.46"
        },
        "2014-07": {
            "credits": 8,
            "gpa": "8.00"
        }
    },
    "credits_registered": 74,
    "credits_earned": 74,
    "cgpa": 8.39,
    "grade_summary": {
        "s": 2,
        "a": 9,
        "b": 11,
        "c": 0,
        "d": 0,
        "e": 1,
        "f": 0,
        "n": 0
    }
```

**Comment by karthikb351** [comment]
+1

@aneesh-neelam: You need to watch for these things, else we'll find them too late

**Comment by kishore-narendran** [comment]
I'll fix them. This would be my bad. 

-----Original Message-----

**Comment by karthikb351** [comment]
That's okay. Still prefer to blame @aneesh-neelam. He's overseeing this project.

**Comment by aneesh-neelam** [comment]
@kishore-narendran: No, I'll fix this. 
@karthikb351: Screw you.

**Comment by sreeram-dev** [comment]
lol.

**Comment by aneesh-neelam** [comment]
Fixed in 99fc5f28d8370def2b0a38f70f8351ca83f6895c.
